### ISSUE TITLE
Unable to verify first certificate

### ISSUE THREAD

**Issue by rgilotra** [necatiarslan/airflow-vscode-extension#19]
I am trying to connect to remote airflow server in vscode. But throw error like  "Unable to verify first certificate". API call  successfully works in postman. Any help is much appreciated.
@necatiarslan 

"System Error !!! h request to {URL}/api/v1/importErrors failed, reason: unable to verify the first certificate"

**Comment by necatiarslan** [comment]
There is a server-side misconfiguration. Here is what I found for this error.

Unfortunately, you are unable to verify the first certificate error because the website or server you are trying to access is using an SSL/TLS certificate that has expired or is no longer valid. This prevents the device or browser from authenticating the certificate.

https://stackoverflow.com/questions/31673587/error-unable-to-verify-the-first-certificate-in-nodejs

I will let you know if I found any other solution.
